lib/streamlit/runtime/secrets.py

```diff
@@ -201,7 +201,7 @@ def _maybe_install_file_watcher(self) -> None:
 
     def _on_secrets_file_changed(self, _) -> None:
         with self._lock:
-            LOGGER.debug(f"Secrets file {self._file_path} changed, reloading")
+            _LOGGER.debug(f"Secrets file {self._file_path} changed, reloading")
```

Comment:
```suggestion
            _LOGGER.debug("Secrets file %s changed, reloading", self._file_path)
```
or
```
LOGGER.debug("Secrets file {file_path} changed, reloading", extra=dict(filepath=self._file_path})
```
I'm not sure CodeQL detects this problem, but we should avoid formatting the strings that we pass to the logger.


In Apache Airflow, we have the following hook:
```
      - id: check-lazy-logging
        name: Check that all logging methods are lazy
        entry: ./scripts/ci/pre_commit/pre_commit_check_lazy_logging.py
        language: python
        pass_filenames: true
        files: \.py$
        exclude: ^airflow/_vendor/
        additional_dependencies: ['rich>=12.4.4', 'astor']
```
https://github.com/apache/airflow/blob/1af25809b1459d16f4a52d2430572cf53c34fe62/.pre-commit-config.yaml#L843-L850
But this check is also available in `pylint`.
https://pylint.pycqa.org/en/latest/user_guide/checkers/features.html#logging-checker